### ISSUE TITLE
Map Data When Running Multiple Instances of Quarm

### ISSUE THREAD

**Issue by OsiViper** [EJWellman/QuarmTool#38]
I love the update for the map data reading from Zeal, however I run into an issue if I have 2 instances running (1 regular character and 1 trader). The map will pull the correct zone from my main character I am playing on, but it will pull the location from my trader, so the zone is correct but location on the map never updates since my trader isn't moving.

**Comment by EJWellman** [comment]
So, it *should* be locking onto which ever one is logged in first. I'll see if I narrow this down, and make the Locked Character work a bit more intelligently.

**Comment by LichLord91** [comment]
yeah it worked for a bit but im having spotty performance on this too.

**Comment by LichLord91** [comment]
If i relaunch the quarm tool and lock it on the right character it works but finicky.

**Comment by EJWellman** [comment]
I had some logic backwards... Changing the locked character will now flip the zeal pipe correctly.
Gonna keep toying with it a bit though.

This'll be in the next release.
